hardware/capture/chipwhisperer-lite/hdl/makeprojects_lin.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
Better to use /bin/sh and not rely on bash as in this case this it not necessary.